features/border-radius.yml

```diff
@@ -0,0 +1,22 @@
+name: border-radius
+description: The `border-radius` CSS property rounds the corners of the border drawn around an element.
+spec: https://drafts.csswg.org/css-backgrounds-3/#corners
+caniuse: border-radius
```

Comment:
Let's `compute_from: css.properties.border-radius` to better align with caniuse.